### PR TITLE
added MP version of second tabview example

### PR DESCRIPTION
**Pull request by uraich**
### Description of the feature or fix

added the MP version of lv_example_tabview_2.c

### Checkpoints
- [ x] Follow the [styling guide](https://github.com/lvgl/lvgl/blob/master/docs/CODING_STYLE.md)
- [ x] Update [CHANGELOG.md](https://github.com/lvgl/lvgl/blob/master/docs/CHANGELOG.md)
- [ ] Update the documentation

### CODE REVIEW

File: examples/widgets/tabview/lv_example_tabview_2.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
+def scroll_begin_event(e):
+
+    #Disable the scroll animations. Triggered when a tab button is clicked */
+    if e.get_code() == lv.EVENT.SCROLL_BEGIN:
+        a = lv.anim_t.cast(e.get_param())
```
Comment (embeddedt) [inline examples/widgets/tabview/lv_example_tabview_2.py]: This needs to be changed to `__cast__` (https://github.com/lvgl/lv_binding_micropython/issues/157#issuecomment-870389807).

Comment (uraich) [reply examples/widgets/tabview/lv_example_tabview_2.py]: In fact, this is what I had in the first place, but I get:
image

Comment (embeddedt) [reply examples/widgets/tabview/lv_example_tabview_2.py]: Your MicroPython version is too old (June 27); the change was made on June 29.

Comment (uraich) [reply examples/widgets/tabview/lv_example_tabview_2.py]: Upgraded my MicroPython version and modified example as you said.

**Comment by embeddedt** [comment]
Thanks!

**Pull request merged**